### ISSUE TITLE
Auto run

### ISSUE THREAD

**Issue by raphaelbastide** [christianalfoni/webpack-bin#156]
I really like the “Auto run JS” option of [JS Bin](http://jsbin.com). It allows the code to be ran automatically during edition. I use it for live design and pedagogic purposes. Did you think about adding such an option in WebpackBin yet?

**Comment by christianalfoni** [comment]
This is a bit more tricky with Webpackbin because it needs to run the code through the server. As I understand JS Bin it runs the code inside a web worker. So not having NPM and webpack support has some advantages :)

The code can still autorun, but it would feel really sluggish compared to JS Bin I believe.. as everything is happening in the client on JS bin. Makes sense?

**Comment by raphaelbastide** [comment]
Yes it totally makes sense. I think the instantaneous aspect of the execution on Bin is not the more important feature: A more slow but regular execution on Webpackbin (e.g. every seconds), only when the user is active, could be awesome for people like me. But as the keyboard shortcuts are quite handy yet, it would be a bonus.

**Comment by FullStackForger** [comment]
Throttled auto-save perhaps. It would be amazing feature!
